BUILD.gn

```diff
@@ -470,6 +470,10 @@ source_set("electron_lib") {
     ]
   }
 
+  if (enable_basic_printing) {
+    deps += [ "//components/services/pdf_compositor:pdf_compositor" ]
```

Comment:
Move this to `chromium_src/BUILD.gn` and also it can be just `//components/services/pdf_compositor`.

In gn `//A/b` is just alias for `//A/b:b` if that target exists.